install/common-pre-install/hpcm_installation-cpi.md

```diff
@@ -0,0 +1,806 @@
+            ```
+
+         1. Add discovered nodes, set image for the discovered nodes.
+
+            ```
+            cm node discover add nodes.conf
+            cinstallman --assign-image --image sles15sp4 --kernel <kernel version> --node '*'
+            cinstallman --set-rootfs tmpfs --node '*'
+            cm node refresh netboot -n '*'
+            ```
+
+            __NOTE:__ Kernel version can be obtained from `cinstallman --show-default-image` command.
+
+         1. Stop the auto-discovery process and power on all nodes.
+
+            ```
+            cm node discover disable
+            cm power status -n '*'
+            cm power on -n '*'
+            ```
+            __NOTE:__ Power on the nodes when 'cm power status -n '*'' lists all the nodes. if there is any errors rerun the command
+            1. Check the status of nodes.
+
+            ```
+            cm power status -n '*'
+            ```
+
+            Example output (expected):
+
+            ```
+            node1        : BOOTED
+            node2        : BOOTED
+            node3        : BOOTED
+            node4        : BOOTED
+            node5        : BOOTED
+            node6        : BOOTED
+            node7        : BOOTED
+            node8        : BOOTED
+            node9        : BOOTED
+            node10       : BOOTED
+            node11       : BOOTED
+            node12       : BOOTED
+            ```
+
+            __NOTE:__ We need to wait at this step till all nodes report status as "BOOTED". Now the ports to all the NCNs can be enabled on switches.
+         1. Create a Bond Network
+	
+## Seed File generation
+
+   Here's the stepwise procedure to generate seedfiles.
+
+   1. Obtain `xnames.csv` file for the targetted system's installation.
+
+      The file will have the following format:
+
+      ```
+      HOSTNAME,RACK,ULOCATION,SUBLOCATION
+      ```
+
+      ```
+       Source,Rack,Location,SUBLOCATION
+       uan01,x3000,u20,
+       ncn-s003,x3000,u09,
+       ncn-s002,x3000,u08,
+       ncn-s001,x3000,u07,
+       ncn-w003,x3000,u06,
+       ncn-w002,x3000,u05,
+       ncn-w001,x3000,u04,
+       ncn-m003,x3000,u03,
+       ncn-m002,x3000,u02,
+       ncn-m001,x3000,u01,
+       nid000004,x3000,u11,L
+       nid000003,x3000,u12,L
+       nid000002,x3000,u11,R
+       nid000001,x3000,u12,R
+       ncn-w004,x3000,u13,
+       sw-spine01,x3000,u39,
+       sw-spine02,x3000,u40,
+       sw-leaf-bmc-01,x3000,u37,
+      ```
+      __NOTE:__ Hostname should be in CSM supported format.
+
+   1. Create `cabinets.yaml` (Manually).

```

Comment:
add a link to this page 
https://github.com/abhishek-g-suresh/docs-csm/blob/main/install/create_cabinets_yaml.md